sonar-db/src/main/java/org/sonar/db/qualityprofile/QualityProfileDao.java

```diff
@@ -351,8 +351,8 @@ public void deleteProjectProfileAssociation(String projectUuid, String profileKe
     mapper(session).deleteProjectProfileAssociation(projectUuid, profileKey);
   }
 
-  public void updateProjectProfileAssociation(String projectUuid, String profileKey, DbSession session) {
-    mapper(session).updateProjectProfileAssociation(projectUuid, profileKey);
+  public void updateProjectProfileAssociation(String projectUuid, String newProfileKey, String oldProfileKey, DbSession session) {
```

Comment:
It tells us that we need to level up our game to help users update their quality profiles!